.internal-ci/test/fs-integration/subaddress-receipt.py

```diff
@@ -0,0 +1,101 @@
+import asyncio
+import sys
+import subprocess
+import json
+import argparse
+repo_root_dir = subprocess.check_output("git rev-parse --show-toplevel", shell=True).decode("utf8").strip()
+sys.path.append("{}/python-library".format(repo_root_dir))
+
+from fullservice import FullServiceAPIv2 as v2
+from basic import wait_for_account_to_sync
```

Comment:
This `wait_for_account_to_sync` is the same as the one in build-submit.py? 
https://github.com/mobilecoinofficial/full-service/blob/pouneh/integ-test/narrative_demo/python-library/build-submit.py#L24

Maybe remove one of them if so?